vimeo/VimeoNetworking
# Fixes SwiftLint script phase

### Pull Request Checklist

### Implementation Summary
Updated the script phase to look in the Pods directory for a copy of `swiftlint` instead of relying on your machine having a copy of SwiftLint installed locally.

### How to Test
- Try putting an opening brace on a new line instead of on the same line.
- Build the project.
- The formatting should auto-correct.
